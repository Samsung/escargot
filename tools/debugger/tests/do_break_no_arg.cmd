b
b
b
b
c
