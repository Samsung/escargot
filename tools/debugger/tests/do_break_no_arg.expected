Connecting to: localhost:6501
Connection created!!!
Stopped at tools/debugger/tests/do_break_no_arg.js:24
(escargot-debugger) b
Error: Argument expected
(escargot-debugger) b
Error: Argument expected
(escargot-debugger) b
Error: Argument expected
(escargot-debugger) b
Error: Argument expected
(escargot-debugger) c
Connection closed.
